scripts/lint-bib.sh

```diff
@@ -5,7 +5,17 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+# Test if there are keys with characters outside alphanumeric, '-', '_', and ':'.
+invalid_keys=$(bibtool --pass.comments=on -- 'select{$key "[^-:A-Za-z0-9_]+"}' docs/references.bib)
+
+if [[ -n "$invalid_keys" ]]; then
+  echo "::error:: There are items in references.bib with keys containing characters" \
+    "outside alphanumeric, '-', '_', and ':':" && echo "$invalid_keys"
```

Comment:
```suggestion
    "outside alphanumeric, '-', '_', and ':':" && printf '%s\n' "$invalid_keys"
```
Maybe this is being excessively paranoid, but I think that this is a safer form of printing.